run/GCClassic/input_options.yml.templates/input_options.yml.fullchem

```diff
@@ -0,0 +1,426 @@
+---
+### GEOS-Chem input_options.yml file for the fullchem simulations
+###
+### NOTE: Add quotes around nitrogen oxide ('NO'), because YAML
+### parsers will confuse this with a negative "no" value.
+
+#============================================================================
+# Simulation settings
+#============================================================================
+simulation:
+  name: ${RUNDIR_SIM_NAME}
+  start_date: [${RUNDIR_SIM_START_DATE}, ${RUNDIR_SIM_START_TIME}]
+  end_date: [${RUNDIR_SIM_END_DATE}, ${RUNDIR_SIM_END_TIME}]
+  root_data_dir: ${RUNDIR_DATA_ROOT}
+  met_field: ${RUNDIR_MET}
+  species_database_file: ./species_database.yml
+  debug_printout: false
+  use_gcclassic_timers: ${RUNDIR_USE_GCCLASSIC_TIMERS}
+
+#============================================================================
+# Grid settings
+#============================================================================
+grid:
+  resolution: ${RUNDIR_GRID_RES_LONG}
+  number_of_levels: ${RUNDIR_GRID_NLEV}
+  longitude:
+    range: ${RUNDIR_GRID_LON_RANGE}
+    center_at_180: ${RUNDIR_CENTER_LON_180}
+  latitude:
+    range: ${RUNDIR_GRID_LAT_RANGE}
+    half_size_polar_boxes: ${RUNDIR_GRID_HALF_POLAR}
+  nested_grid_simulation:
+    activate: ${RUNDIR_GRID_NESTED_SIM}
+    buffer_zone_NSEW: ${RUNDIR_GRID_BUFFER_ZONE}
+
+#============================================================================
+# Timesteps settings
+#============================================================================
+timesteps:
+  transport_timestep_in_s: ${RUNDIR_TRANSPORT_TS}
+  chemistry_timestep_in_s: ${RUNDIR_CHEMISTRY_TS}
+  radiation_timestep_in_s: 10800
+
+#============================================================================
+# Settings for GEOS-Chem operations
+#============================================================================
+operations:
+
+  chemistry:
+    activate: true
+    linear_chemistry_aloft:
+      activate: true
+      use_linoz_for_O3: true
+    active_strat_H2O:
+      activate: true
+      use_static_bnd_cond: true
+    gamma_HO2: 0.2
+
+  convection:
+    activate: true
+
+  dry_deposition:
+    activate: true
+    CO2_effect:
+      activate: false
+      CO2_level: 600.0
+      reference_CO2_level: 380.0
+    diag_alt_above_sfc_in_m: 10
+
+  pbl_mixing:
+     activate: true
+     use_non_local_pbl: ${RUNDIR_USE_NLPBL}
+
+  photolysis:
+    input_directory: ${RUNDIR_DATA_ROOT}/CHEM_INPUTS/FAST_JX/v2021-10/
+    overhead_O3:
+      use_online_O3_from_model: ${RUNDIR_USE_ONLINE_O3}
+      use_column_O3_from_met: true
+      use_TOMS_SBUV_O3: false
+    photolyze_nitrate_aerosol:
+      activate: false
+      NITs_Jscale_JHNO3: 0.0
+      NIT_Jscale_JHNO2: 0.0
+      percent_channel_A_HONO: 66.667
+      percent_channel_B_NO2: 33.333
+
+  rrtmg:
```

Comment:
@laestrada, RRTMG is the radiative transfer model that a lot of the aerosols people use.  It can provide radiative forcing fluxes as output (although it adds a lot of overhead).  I can make that "rrtmg_rad_forcing" or something clearer if you wish.

Reply:
Changed the tag to `rrtmg_rad_transfer_model`

Reply:
Also fixed in a6cc6617a6ecb9597e8d39ce7ceff0eb880739e4